### DISCUSSION TITLE
Is there a way to verify the released binary?

### DISCUSSION THREAD

**Discussion by bcelary** [doublecmd/doublecmd#778] [General]
I am not able to find a way to verify integrity or authenticity of the binary package. Did I miss a place that contains this information?

**Comment by alexx2000** [comment]
No such information.

**Reply by bcelary** [reply]
could this be worth adding to have some idea if binaries were tampered with?

**Comment by goyalyashpal** [comment]
if wanted, this may help:

* https://github.com/wez/wezterm/issues/2114#issuecomment-1194872432
* https://github.com/wez/wezterm/commit/360b3f347902d610ba307b41c40ff8a5cafb37ba
